pkg/local/authorization.go

```diff
@@ -1,22 +1,49 @@
 package local
 
 import (
+	"encoding/json"
 	"net/http"
+	"strings"
 
 	"go.uber.org/zap"
 
 	"github.com/cmsgov/easi-app/pkg/appcontext"
 	"github.com/cmsgov/easi-app/pkg/authn"
 )
 
-// If you're developing interfaces with LDAP, you may need to change this to your own EUA ID for testing purposes
-const testEUAID = "ABCD"
+// DevUserConfig is the set of values that can be passed in a request header
+type DevUserConfig struct {
+	EUA      string   `json:"eua"`
+	JobCodes []string `json:"jobCodes"`
+}
 
 func authorizeMiddleware(logger *zap.Logger, next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		logger.Info("Using local authorization middleware and populating EUA ID")
-		ctx := appcontext.WithPrincipal(r.Context(), &authn.EUAPrincipal{EUAID: testEUAID, JobCodeEASi: true})
-		next.ServeHTTP(w, r.WithContext(ctx))
+		logger.Info("Using local authorization middleware")
+
+		if len(r.Header["Authorization"]) == 0 {
+			logger.Info("No Authentication header present")
+			next.ServeHTTP(w, r)
+		} else {
+			tokenParts := strings.Split(r.Header["Authorization"][0], "Bearer ")
+			if len(tokenParts) < 2 {
+				logger.Fatal("invalid Bearer in auth header")
```

Comment:
Is `Fatal` really what you want to do here? I believe that ends up calling `os.Exit(1)`, killing the entire server.